web/themes/custom/news/dist/main.css

```diff
@@ -420,10 +420,10 @@ a, .link {
       color: #000000; }
 
```

Comment:
@TerryKolodii 
We don't commit dist folder contents

it should be in .gitignore

Reply:
fixed

Reply:
@TerryKolodii you need to delete the file, then commit it.

Afterwards add it to gitignore.

Spoiler alert: maybe adding it to .gitignore should be one with a new PR, we'
ll see

Reply:
> Spoiler alert: maybe adding it to .gitignore should be one with a new PR, we' ll see
do you mean I need to do that with my next PR? Not in this one?

Reply:
> Spoiler alert: maybe adding it to .gitignore should be one with a new PR, we'
> ll see

Do you mean I need to do that with  my next PR? Not in this one?